modules/olm-configuring-aws-sts.adoc

```diff
@@ -0,0 +1,138 @@
+
+:_content-type: PROCEDURE
+[id="efs-sts_{context}"]
+= Configuring an Operator to use Amazon Web Services Secure Token Service
+
+You can configure Operators on Amazon Web Services (AWS) clusters to use the AWS Secure Token Service (STS). Operators that use STS assign components IAM roles that provide short-term, limited-privilege security credentials.
+
+Perform this procedure after installing the an Operator that can use STS, but before installing any required drivers for the Operator. If you perform this procedure after installing a driver for an Operator, the Operator might not be able to function correctly.
+
+[NOTE]
+====
+By default, `ccoctl` creates objects in the directory in which the commands are run. To create the objects in a different directory, use the `--output-dir` flag. This procedure uses `<path_to_ccoctl_output_dir>` to refer to this directory.
+
+Some `ccoctl` commands make AWS API calls to create or modify AWS resources. You can use the `--dry-run` flag to avoid making API calls. Using this flag creates JSON files on the local file system instead. You can review and modify the JSON files and then apply them with the AWS CLI tool using the `--cli-input-json` parameters.
+====
+
+.Prerequisites
+
+* You have an AWS cluster that is configured to use STS.
+
+* You have the {product-title} release image that was used to install the cluster.
+
+* You have extracted and prepared the `ccoctl` binary during cluster installation. For more information, see "Configuring the Cloud Credential Operator utility".
+
+.Procedure
+
+. Determine the name of the `CredentialsRequest` custom resource (CR) in the `openshift-cloud-credential-operator` namespace for the Operator:
+//How do they do that?
+
+. Extract the CR to the `credrequests` directory by running the following command:
++
+[source,terminal]
+----
+$ oc get credentialsrequest \
+  -n openshift-cloud-credential-operator <custom_resource_name> \
+  -o yaml > <path_to_directory_with_list_of_credentials_requests>/credrequests/<custom_resource_name>.yaml
+----
+
+//Existing CSI Operator has an example CR, but the CR extraction should create a YAML file that is correct without the user needing to mess with it, right?
+////
+. Create and save an EFS `CredentialsRequest` YAML file, such as shown in the following example, and then place it in the `credrequests` directory:
++
+.Example
++
+[source, yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-aws-efs-csi-driver
+  namespace: openshift-cloud-credential-operator
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries:
+    - action:
+      - elasticfilesystem:*
+      effect: Allow
+      resource: '*'
+  secretRef:
+    name: aws-efs-cloud-credentials
+    namespace: openshift-cluster-csi-drivers
+  serviceAccountNames:
+  - aws-efs-csi-driver-operator
+  - aws-efs-csi-driver-controller-sa
+----
+////
```

Comment:
I am leaning towards leaving this out - iiuc the user doesn't need to edit this, so it's simpler imo not to include it.